tests/unit_tests/udf_wrapper_params/zero_shot/error_on_prediction_multiple_model_multiple_batch.py

```diff
@@ -1,29 +1,9 @@
 from pathlib import PurePosixPath
 from exasol_udf_mock_python.connection import Connection
-from tests.unit_tests.udf_wrapper_params.zero_shot.mock_zero_shot import \
-    MockZeroShotFactory, MockZeroShotModel, MockPipeline
-
-
-def udf_wrapper():
-    from exasol_udf_mock_python.udf_context import UDFContext
-    from exasol_transformers_extension.udfs.models.\
-        zero_shot_text_classification_udf import ZeroShotTextClassificationUDF
-    from tests.unit_tests.udf_wrapper_params.zero_shot.mock_sequence_tokenizer \
-        import MockSequenceTokenizer
-    from tests.unit_tests.udf_wrapper_params.zero_shot.\
-        error_on_prediction_multiple_model_multiple_batch import \
-        ErrorOnPredictionMultipleModelMultipleBatch as params
-
-    udf = ZeroShotTextClassificationUDF(
-        exa,
-        batch_size=params.batch_size,
-        pipeline=params.mock_pipeline,
-        base_model=params.mock_factory,
-        tokenizer=MockSequenceTokenizer)
-
-    def run(ctx: UDFContext):
-        udf.run(ctx)
 
+from tests.unit_tests.udf_wrapper_params.zero_shot.make_data_row_functions import make_number_of_strings, sub_dir, \
+    model_name, text_data, label, make_input_row, score, make_output_row, make_model_output_for_one_input_row, \
+    bucketfs_conn
```

Comment:
I think the continuation using parentheses makes the code more readable, e.g.
```
from ... import (make_number_of_strings, sub_dir,
    model_name, text_data, label, ...)
```

Reply:
will change stuff like this once the linter is in the project